### DISCUSSION TITLE
EXA_COLORS

### DISCUSSION THREAD

**Discussion by andrei1015** [eza-community/eza#229] [Q&A, Howto, Help] [answered]
Hello!

I just changed to eza, thank you for continuing!
With exa I had some settings in the EXA_COLORS environment variable, which still seems to work. Would be an idea to leave compatibility for that one as well, but also ad support for EZA_COLORS maybe? Or maybe I am getting something wrong? I tried with EZA_COLORS but for example the user and group columns seem to not be recolored

**Comment by PThorpe92** [comment, marked as answer]
Currently we are backwards compatible for `EXA_COLORS`and support LS_COLORS as well for basic coloring options.

I believe it's mainly important to make sure that we do not break anyone's existing ENV vars, and that the alias only has to change one letter to continue working :smile: 
We very well may in the future, add support for EZA_COLORS, as the new name gets recognized more and people are used to it.
